### PR TITLE
View Source shortcut should be Ctrl+U on Windows, Cmd+Alt+U on macOS

### LINKED ISSUE #7702: View Source shortcut should be CTRL+U, not CTRL+ALT+U
## Test plan
1. Launch Brave on macOS and visit brave.com
2. Verify `view source` shortcut is unchanged; it should be Cmd + Option + U
3. Launch Brave on Windows (or Linux) and visit brave.com
4. Press Ctrl + U and verify `view source` comes up as expected

## Original issue description
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
View Source shortcut should be CTRL+U, not CTRL+ALT+U

- Platform (Win7, 8, 10? macOS? Linux distro?):
All

- Brave Version (revision SHA):
1db81cb18b6919535792bb7e198ab87276bd67ea

- Steps to reproduce:
    1. Go to //brave.com
    2. Press CTRL+U (win) or CMD+U (darwin)
    
- Actual result:
Nothing happens

- Expected result:
URI view-source:https://brave.com is loaded in a new tab
The context-menu option to View Source indicates that the shortcut should be CTRL+U

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
Yes

- Can this issue be consistently reproduced?
Yes

- Any related issues:
Referenced in, but not related to #7491. View-Source shortcut still doesn't work even after commit 5a889ccd6198906537f0c29d4b28067292a4ef78 (but that commit is necessary for any keybindings to work)

### PR DESCRIPTION
**Pull request by mrpapercut**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Ran `git rebase -i` to squash commits (if needed).

View Source shortcut should be Ctrl+U on Windows, Cmd+Alt+U on macOS

Fixes #7702 (see discussion)

### CODE REVIEW

File: app/localShortcuts.js [hunk 1]
```diff
@@ -35,17 +34,24 @@ module.exports.register = (win) => {
...
     // We disable for development mode because Browser level dev tools copy doesn't work.
     // Workaround for #1060
-    simpleWebContentEvents.push([
-      'Cmd+C',
-      messages.SHORTCUT_ACTIVE_FRAME_COPY
-    ])
+    simpleWebContentEvents.push(
+      ['Cmd+C', messages.SHORTCUT_ACTIVE_FRAME_COPY]
+    )
+  } else {
```
Comment (NejcZdovc) [inline app/localShortcuts.js]: you have in a previous else if `(process.env.NODE_ENV !== 'development')` , that means that your else would only trigger if you are in `development` and in `darwin`

Comment (mrpapercut) [inline app/localShortcuts.js]: Oh boy do I feel stupid. Yes, absolutely right, should've been more careful. Rev 770596a adds Cmd+Alt+U for darwin always, and adds the Cmd+C only if not in dev-mode

**Comment by bsclifton** [comment]
@NejcZdovc looks like @mrpapercut made the changes; can you re-review and merge if OK? Thanks 😄

**Comment by NejcZdovc** [comment]
@bsclifton can you confirm that is working correctly on Windows?

**Pull request merged**